apps/api-gateway/src/user/user.service.ts

```diff
@@ -9,16 +9,18 @@ import { UpdateUserProfileDto } from './dto/update-user-profile.dto';
 import { AddPasskeyDetails } from './dto/add-user.dto';
 import { UpdatePlatformSettingsDto } from './dto/update-platform-settings.dto';
 import { CreateUserCertificateDto } from './dto/share-certificate.dto';
+import { UsersProfile } from 'apps/user/interfaces/user.interface';
```

Comment:
Let's have consistency in interface names. We have decided to use "I" prefix and then interface name.

Reply:
Resolved.